genesisl1/chains.json

```diff
@@ -0,0 +1,60 @@
+  
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "genesisl1",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "GenesisL1",
+  "chain_id": "genesis_29-2",
+  "bech32_prefix": "genesis",
```

Comment:
Please register this prefix here:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md